pkg/controller/secret/secret_controller_test.go

```diff
@@ -297,13 +299,13 @@ func Test_createPagerdutyRoute(t *testing.T) {
 }
 
 func Test_createPagerdutyReceivers_WithoutKey(t *testing.T) {
-	assertEquals(t, 0, len(createPagerdutyReceivers("", "")), "Number of Receivers")
+	assertEquals(t, 0, len(createPagerdutyReceivers("", "", "fake-cluster-id")), "Number of Receivers")
 }
 
 func Test_createPagerdutyReceivers_WithKey(t *testing.T) {
 	key := "abcdefg1234567890"
 
-	receivers := createPagerdutyReceivers(key, exampleConsoleUrl)
+	receivers := createPagerdutyReceivers(key, exampleConsoleUrl, "fake-cluster-id")
```

Comment:
Can we make `"fake-cluster-id"` a const, and then use it in all these places? This just ensures that the test and the example string always match